src/components/SignUp.tsx

```diff
@@ -64,6 +66,14 @@ const SignUp = () => {
         emailRef.current!.value,
         passwordRef.current!.value
       )
+      .then((cred) => {
+        if (cred.user != null) {
+          return db.collection("user").doc(cred.user.uid).set({
```

Comment:
Changed name of collection from "users" to "user" as the default authentication collection has the name "users".